worker-jobtracking/src/main/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerFactory.java

```diff
@@ -389,4 +394,172 @@ private void forwardAvailableJobs(final List<JobTrackingWorkerDependency> jobDep
             LOG.error("Error retrieving Dependent Job Info from the Job Service Database {}", e);
         }
     }
+
+    @Override
+    public void processTasks(final BulkWorkerRuntime bwr) throws InterruptedException
+    {
+        // TODO should come from configuration
+        // Calculate a batch timeframe
+        // Let's say 10 seconds
+        final long cutoffTime = System.currentTimeMillis() + 10000;
+        final List<WorkerTask> workerTasksThatWillNeedToGoToDatabase = new ArrayList<>();
+        String partition = "";
```

Comment:
I want this current partition / job concept gone.  It will not provide performance benefits when tasks are heavily interleaved.  Instead we want to be building up a collection keyed on the partition id / job id key.